services/authors/src/server.js

```diff
@@ -7,7 +7,7 @@ const EXPRESS_PORT = process.env.EXPRESS_PORT || 8080;
 
 (async () => {
   try {
-    await db.authenticate();
+    await db.sync({ force: true });
```

Comment:
Why do you enable force: true? This will clear all migrations inside the microservice, every time you launch an application instance, you need to leave the previous version
```suggestion
    await db.authenticate();
```